src/libraries/System.Private.CoreLib/src/System/SpanHelpers.Byte.cs

```diff
@@ -1069,6 +1079,22 @@ private static int LocateLastFoundByte(Vector<byte> match)
             return i * 8 + LocateLastFoundByte(candidate);
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private static Vector<byte> LoadVector(ref byte start, nuint offset)
```

Comment:
nit: this helper is not needed because there is `Vector.LoadUnsafe`

Reply:
I've not changed any code.  I only moved a few methods in the source to reduce the number of ifdefs.